src/data-access/repositories/mongo/mongo-ingested-part-repository.ts

```diff
@@ -32,12 +32,12 @@ export class MongoIngestedPartRepository extends BaseMongoRepository implements
     }
     return {
       ...this.mongoIngestedEntityConverter.convertToIngestedPart(mongoIngestedPart),
-      ingestedPieces: await this.ingestedPieceRepository.getIngestedPieces(mongoIngestedPart._id)
+      ingestedPieces: await this.ingestedPieceRepository.getIngestedPiecesByPart(mongoIngestedPart._id)
     }
   }
 
-  public async getIngestedParts(segmentId: string): Promise<IngestedPart[]> {
-    this.assertDatabaseConnection(this.getIngestedParts.name)
+  public async getIngestedPartsBySegment(segmentId: string): Promise<IngestedPart[]> {
```

Comment:
I am glad for the change to add the scope of the request, but is there a difference between `BySegment` and `ForSegment`? We use currently use both.
It would be nice to only use one or the another (unless the meaning differs).